src/modules/75-ci/components/PipelineStudio/BuildStageSpecifications/BuildStageSpecifications.tsx

```diff
@@ -216,7 +216,7 @@ export default function BuildStageSpecifications({ children }: React.PropsWithCh
           formName="ciBuildStage"
           onSubmit={values => logger.info(JSON.stringify(values))}
         >
-          {formik => {
+          {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
          {(formik: unknown) => {
```